static_src/components/disclaimer.jsx

```diff
@@ -1,33 +1,102 @@
-
-import classNames from 'classnames';
-import style from 'cloudgov-style/css/cloudgov-style.css';
+import classnames from 'classnames';
 import React from 'react';
-
-import createStyler from '../util/create_styler';
 import { config } from 'skin';
 
-export default class Disclaimer extends React.Component {
+const arrowUpImg = require('../img/angle-arrow-down-primary.svg');
+const arrowDownImg = require('../img/angle-arrow-up-primary-hover.svg');
+const dotGovIcon = require('../img/icon-dot-gov.svg'); // require('cloudgov-style/img/icon-dot-gov.svg');
+const httpsIcon = require('../img/icon-https.svg'); //require('cloudgov-style/img/icon-https.svg');
+const flag = require('cloudgov-style/img/us_flag_small.png');
+
+const dotGovAlt = 'Dot gov';
+const httpsAlt = 'Https';
+const flagAlt = 'US flag signifying that this is a United States Federal Government website';
 
+const CONTROLS = 'gov-banner';
+
+export default class Disclaimer extends React.Component {
   constructor(props) {
     super(props);
-    this.styler = createStyler(style);
+
+    this.state = {
+      expanded: false
+    };
+
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(event) {
+    event.preventDefault();
+
+    this.setState({
+      expanded: !this.state.expanded
+    });
   }
 
   render() {
+    const { expanded } = this.state;
+    const hidden = !expanded;
+    const panelClass = classnames('usa-banner-content usa-grid usa-accordion-content', {
+      hide: hidden
+    });
+    const bannerBarClass = classnames('usa-banner usa-disclaimer disclaimer-no_sidebar', {
```

Comment:
One question, why do we need the expanded class here?

Reply:
That was initially taking care of the chevon.